.rubocop.yml

```diff
@@ -9,10 +9,21 @@ AllCops:
 
 # Cop's to ignore
 
+
+# It thinks that Ruby version dependend definitions are duplicates
+Bundler/DuplicatedGem:
+  Exclude:
+    - "Gemfile"
+
 # With this enabled it suggests a change that will break the Gemfile
 Lint/AssignmentInCondition:
   Enabled: false
 
+# It would need a complete rewrite of the test to get this cop satisfied
+Metrics/BlockLength:
+  Exclude:
+    - "spec/**/*.rb"
+
```

Comment:
see https://github.com/bbatsov/rubocop/issues/3772 if interessted in more details